src/AddTodoForm.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import InputWithLabel from "./InputWithLable";
+
+function AddTodoForm(props) {
+
+    const [todoTitle, setTodoTitle] = useState('')
+    
+    //sets the current state of the input field
+    const handleTitleChange = (e) => {
+        const newTodoTitle = e.target.value;
+
+        setTodoTitle(newTodoTitle);
+    }
+
+    //After the form gets submitted we set our form to an empty array
+    const handleAddTodo = (e) => {
+        e.preventDefault();
+        setTodoTitle('');
+
+       props.onAddTodo({
+           title: todoTitle,
+           id: Date.now()
+       }); 
+    }
+  
+    return(
+        <form onSubmit={handleAddTodo}>
+            <InputWithLabel onChange={handleTitleChange} value={todoTitle} label="Title"/>
```

Comment:
This is what I was referencing below.

Reply:
If you look at the 2nd video about 1min and 35 seconds in you'll see what using children in InputWithLabel looks like https://learn.codethedream.org/react-fundamentals-reusable-components-imperative-react/